### PR TITLE
Firstarticlebalresch

### PR DESCRIPTION
**Pull request by balresch**
added short 'slugs' info to glossary

### CODE REVIEW

File: src/docs/glossary/index.md [hunk 1]
```diff
@@ -4,3 +4,4 @@ author: Santiment Team
 ---
 
 - [Asset](/glossary/asset/)
+- [slug](/glossary/slug/)
```
Comment (Partysun) [inline src/docs/glossary/index.md]: @balresch Slug should be with a capital letter.

File: src/docs/glossary/slug/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: Glossary - slug
+author: Santiment Team
+---
+
+# slug
```
Comment (Partysun) [inline src/docs/glossary/slug/index.md]: and here

File: src/docs/navigation.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ export const CATEGORIES = [
     title: "Glossary",
     icon: Glossary,
     description: "Introduction to all relevant terms and concepts",
-    articles: ["Asset"]
+    articles: ["Asset", "slug"]
```
Comment (Partysun) [inline src/docs/navigation.js]: and here

Comment (balresch) [reply src/docs/navigation.js]: should be good now :)

**Pull request merged**